### PR TITLE
F/abaird hypothermia

### PR DESCRIPTION
**Pull request by ajbaird**
- added caffeine substance 
- updated heat models to include clo 
- added validation scenario of changing clo 
- updated shivering metabolic increases and hypothermic energy decline (neither were well validated and actually broke biogears in some ways during hypothermia)
- included new patient events for mild, moderate, and severe hypothermia

### CODE REVIEW

File: projects/biogears/libBiogears/src/engine/Systems/Environment.cpp [hunk 1]
```diff
@@ -745,7 +752,7 @@ void Environment::CalculateEvaporation()
 
     for (SEThermalCircuitPath* envSkinToGround : m_EnvironmentSkinToGroundPaths) {
       double dClothingResistance_clo = GetConditions().GetClothingResistance(HeatResistanceAreaUnit::clo) * m_cloSegmentation[index];
-      double dClothingResistance_m2_kPa_Per_W = clo_To_m2_K_Per_W * dClothingResistance_clo / (iCl * dLewisRelation_K_Per_kPa);
+      double dClothingResistance_m2_kPa_Per_W = 100000*clo_To_m2_K_Per_W * dClothingResistance_clo / (iCl * dLewisRelation_K_Per_kPa);
```
Comment (StevenAWhite) [inline projects/biogears/libBiogears/src/engine/Systems/Environment.cpp:755]: Major multiplier batman. I have this very basic request that we start using digit separators.  `100_000`

File: share/xsd/cdm/Patient.xsd [hunk 2]
```diff
@@ -113,8 +113,12 @@ specific language governing permissions and limitations under the License.
       <!--<< @brief Very low glucose in the bloodstream (< 50 mg/dL). -->
       <xs:enumeration value="HypoglycemicComa"/>
       <!--<< @brief Dangerously low glucose in the bloodstream (< 20 mg/dL). -->
-      <xs:enumeration value="Hypothermia"/>
-      <!--<< @brief The condition of having a body temperature greatly below normal (< 35° C). -->
+      <xs:enumeration value="MildHypothermia"/>
```
Comment (StevenAWhite) [inline share/xsd/cdm/Patient.xsd:116]: Might be a enum in PatientEnums.h that needs to be updated

File: projects/biogears/libBiogears/src/engine/Systems/Environment.cpp (hunk 1 above)
Comment (ajbaird) [reply projects/biogears/libBiogears/src/engine/Systems/Environment.cpp:755]: yeah this is sort of a shallow patch for now to have something happen due to clothing change. biogears currently doesn't do pretty much anything when you take all your clothes off

Comment (ajbaird) [reply projects/biogears/libBiogears/src/engine/Systems/Environment.cpp:755]: i'll be working more on hypothermia over the coming 6 months so I'm hoping to update the models to be a bit more robust lol

File: share/xsd/cdm/Patient.xsd (hunk 2 above)
Comment (ajbaird) [reply share/xsd/cdm/Patient.xsd:116]: I'll double check

**Comment by StevenAWhite** [comment]
Merged

**Pull request merged**